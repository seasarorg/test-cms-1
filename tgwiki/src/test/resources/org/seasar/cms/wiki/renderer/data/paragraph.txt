* Tuigwaa のライセンスについて

|hoge|日本語|三つ目|四つ目|
|さあどうなる|これも|テーブル|だよ|

** 子要素

>引用してみる。
引用続いている。
>>二段目の引用。さらに引用する?
>>>三段目の引用

----

*** 孫要素 [#tg00]

何かあったらhogehoge@hogehoge.comまでご連絡ください。

* 次。

[[アンカー>#tg00]]です。