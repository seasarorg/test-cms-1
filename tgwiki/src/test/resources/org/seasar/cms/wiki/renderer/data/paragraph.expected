<h2>
 Tuigwaa のライセンスについて
</h2>
<table>
<tbody>
<tr><td>
hoge
</td>
<td>
日本語
</td>
<td>
三つ目
</td>
<td>
四つ目
</td>
</tr><tr><td>
さあどうなる
</td>
<td>
これも
</td>
<td>
テーブル
</td>
<td>
だよ
</td>
</tr></tbody>
</table>
<h3>
 子要素
</h3>
<blockquote>引用してみる。引用続いている。<blockquote>二段目の引用。さらに引用する?<blockquote>三段目の引用</blockquote>
</blockquote>
</blockquote>
<hr/>
<h4>
 孫要素 <a id="tg00" class="anchor_super" href="#tg00">&nbsp;</a>
</h4>
<p>何かあったらhogehoge@hogehoge.comまでご連絡ください。</p>
<h2>
 次。
</h2>
<p><a href="#tg00">アンカー</a>です。</p>
